docs/modular-resources/advanced/create-subtype.md

```diff
@@ -4,76 +4,83 @@ linkTitle: "New API Subtype"
-description: "Define a new resource subtype using an existing resource API."
+description: "Define a new resource subtype and API for a resource that does not fit into existing component or service subtypes."
 no_list: true
 aliases:
   - "/extend/modular-resources/create/create-subtype/"
 ---
 
-{{% alert title="Tip" color="tip" %}}
+You can define a new resource _{{< glossary_tooltip term_id="subtype" text="subtype" >}}_ (and an API for that subtype) if:
 
-[Modular resources](/modular-resources/) provided by custom {{< glossary_tooltip term_id="module" text="modules" >}} are the preferred method of creating custom resource implementations.
-
-This option is provided for advanced users who are unable to use the modular resource system with the robot deployments.
-
-{{% /alert %}}
-
-You need to define a new resource _subtype_ (and an API for that subtype) if:
-
-- You have a {{% glossary_tooltip term_id="resource" text="resource" %}} that does not fit into any of the existing [component](/components/) or [service](/services/) subtypes.
+- You have a {{% glossary_tooltip term_id="resource" text="resource" %}} that does not fit into any of the existing {{< glossary_tooltip term_id="component" text="component" >}} or {{< glossary_tooltip term_id="service" text="service" >}} subtypes.
 - You have a resource that could fit into an existing subtype, but you want to define an API with different methods and messages than the ones in the existing [APIs](/program/apis/) for that subtype.
   For example, you have a [sensor](/components/sensor/) and you want to define a `Calibrate` method.
 
-{{% alert title="Tip" color="tip" %}}
+  {{% alert title="Tip" color="tip" %}}
 
-If you want to use most of an existing API but need just a few other functions, try using the [`DoCommand`](/program/apis/#docommand) endpoint and [extra parameters](/program/use-extra-params/) to add custom functionality to an existing subtype.
+  If you want to use most of an existing API but need just a few other functions, try using the [`DoCommand`](/program/apis/#docommand) endpoint and [extra parameters](/program/use-extra-params/) to add custom functionality to an existing subtype.
 
-{{% /alert %}}
+  {{% /alert %}}
 
 ## Define your new resource API
 
 Viam uses [protocol buffers](https://protobuf.dev/) for API definition.
 
-To define a new subtype, you need to define the methods and messages of the new API in [protobuf](https://github.com/protocolbuffers/protobuf), write code in Python or Go to implement the higher level server and client functions required, and generate all necessary [protobuf module files](https://buf.build/docs/generate/usage/).
+To define a new subtype, you need to define the methods and messages of the new API in [protobuf](https://github.com/protocolbuffers/protobuf), generate all necessary [protobuf module files](https://buf.build/docs/generate/usage/), and write code in Python or Go to implement the higher level server and client functions required.
```

Comment:
Why this change? I'd do it in the originally-written order I think. I *think* it'd be easier to figure out what you need by writing the actual code, and then making the protobuf to fit later. But maybe either is possible? @mcvella